_data-prepper/pipelines/configuration/processors/parse-json-processor.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Parse JSON processor
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# Parse JSON processor
+
+The `parse_json` processor parses JSON data for an event, including any nested fields. 
```

Comment:
When we say "parse" here, can we get some more details on what is specifically happening? Is the data being extracted? Transformed? Etc.

Reply:
According to the examples given below, the source/pointer json data will be extracted and the input event will be added with the extracted fields.

Reply:
@chenqi0805 How about this: `The `parse_json` processor parses JSON data for an event, including any nested fields. The processor extracts the JSON pointer data and adds the input event to the extracted fields.`?